iron-overlay-behavior.html

```diff
@@ -155,11 +155,8 @@
       return Polymer.dom(this).querySelector('[autofocus]') || this;
     },
 
-    registered: function() {
-      this._backdrop = document.createElement('iron-overlay-backdrop');
-    },
-
     ready: function() {
+      this._backdrop = this._manager.getBackdropInstance();
```

Comment:
you could completely remove `this._backdrop` from here, and just return `this.manager.getBackdropInstance()` in `get backdropElement()`